deprecated/warp/src/constellation.rs

```diff
@@ -0,0 +1,393 @@
+#[cfg(not(target_arch = "wasm32"))]
+pub mod ffi {
```

Comment:
`deprecation` implies the API still part of the lib and available, just marked as "soon to be removed". AFAICT, you're having this code be completely ignored by the compiler.

With git remembering everything (unless we rewrite history on purpose and still `git reflog` remembers locally), I really don't see any value in keeping dead code around. It only confuses people new to the code. If the dead code is needed in the future, it can very easily be revived from git history.

So I strongly recommend we just delete this all.

Reply:
While internally we could just look at the history (unlikely for one to to rewrite it anyway), I dont see any reason to delete it all right at this moment since it can still be easily be referenced (although `warp-bin` technically outlived its stay there), and given the location of the code its unlikely for a newcomer to get confused per se (since many people have ignored that directory when looking at the code for the first time), however I may rename the `deprecated` directory.

Reply:
I went on and removed it and will add a issue to reference back to the commit with them later when we do need to add FFI code again.